roles/cassandra-tools/tasks/main.yml

```diff
@@ -0,0 +1,14 @@
+---
+- name: Cassandra Tools | Add datastax APT repository keys
```

Comment:
@mijicd do we need to pull it using apt? We can download zip and untar and this is enough (so only two tasks, curl or wget and that unzip).

Here is the [link](https://docs.datastax.com/en/cassandra/2.0/cassandra/install/installTarball_t.html) to install from tarball.